V2 : changes to standardize the publishing of core packages

z3dev opened the pull request:
There are lots of ways to publish packages, however some standards are emerging for UMD and ES6 packages. As such, JSCAD packages need to adhere to some of the standards now, which will increase the usability of the packages, and increase the adoption of JSCAD.

Here's a summary of what changed:
- syncronized .gitignore and .npmignore so unwanted contets are not commited / published
- standardized package contents; descriptions, keywords, repository, etc
- standardized publishing (core/modeling/web/regl-renderer)
  - standard build scripts
  - standard library names
  - added "unpkg" setting for direct import of the pre-built libraries (UMD)
- HTML documents changed to use the NEW library names

Here's some good references
- www.unpkg.com
- https://www.freecodecamp.org/news/anatomy-of-js-module-systems-and-building-libraries-fadcd8dbd0e/

**NOTE: NO LOGIC CHANGES WERE MADE.**

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Does your submission pass tests?

SimonClark commented on packages/examples/package.json:
```diff
@@ -1,13 +1,12 @@
     "release-minor": "git checkout master && git pull origin master && npm version minor",
-    "release-major": "git checkout master && git pull origin master && npm version major",
-    "postinstall": "node -e \"console.log('\\u001b[35m\\u001b[1mLove OpenJSCAD? You can now donate to our open collective:\\u001b[22m\\u001b[39m\\n > \\u001b[34mhttps://opencollective.com/openjscad/donate\\u001b[0m')\""
+    "release-major": "git checkout master && git pull origin master && npm version major"
```
Won't this get V1?  Use V2 instead of master?

z3dev replied:
Good point. Actually, V2 doesn’t use those NPM commands currently, so I will drop those for now, and remove any kind of doubts.

SimonClark reviewed (commented):
one quick question, otherwise, looks good.

z3dev commented:
@SimonClark thanks for the review.

i updated the packages, and removed the release-* scripts. i'm not sure those are necessary, but i could be wrong.
anyway, for now, those scripts are not required.
